controller-server/src/main/java/com/yahoo/vespa/hosted/controller/role/PathGroup.java

```diff
@@ -4,8 +4,10 @@
 import com.yahoo.restapi.Path;
 
 import java.util.EnumSet;
+import java.util.List;
 import java.util.Optional;
 import java.util.Set;
+import java.util.regex.Pattern;
```

Comment:
Looks to be unused (hopefully)?

Reply:
Moved to the unit test — used to `quote(...)` the `{tenant}` et al when searching for them in the path specs.